vitest.config.ts

```diff
@@ -16,6 +16,10 @@ const projectsWithTests: { scope?: string; names: string[] }[] = [
 		scope: 'studiocms',
 		names: ['devapps'],
 	},
+	{
+		scope: 'studiocms',
+		names: ['html'],
+	},
```

Comment:
```suggestion
		names: ['devapps', 'html'],
	}
```

This was designed for this :sweat_smile: